apps/api/Dockerfile

```diff
@@ -1,11 +1,10 @@
 FROM node:20-alpine3.19 as dev_base
 RUN apk add g++ make py3-pip
```

Comment:
we can exclude cache because we use dev_base in the final step `RUN apk add --no-cache g++ make py3-pip`